zerver/lib/integrations.py

```diff
@@ -83,6 +83,14 @@ def url_object(self):
     WebhookIntegration('deskdotcom', logo='static/images/integrations/logos/deskcom.png', display_name='Desk.com'),
     WebhookIntegration('freshdesk'),
     WebhookIntegration('github', function='zerver.views.webhooks.github.api_github_landing', display_name='GitHub'),
+    WebhookIntegration(
```

Comment:
We should probably add a `secondary_line_text` for the old GitHub integration that's e.g. `(deprecated)`, and a note at the top of its description that it's using the old "GitHub services" system which is deprecated, maybe with a link to https://github.com/github/github-services.